tools/frr-reload.py

```diff
@@ -715,38 +736,6 @@ def line_exist(lines, target_ctx_keys, target_line, exact_match=True):
     return False
 
 
-def check_for_exit_vrf(lines_to_add, lines_to_del):
```

Comment:
exit-vrf is the right syntax used in running-config. Rather than removing `exit-vrf`, can `exit` line skip this block? That more aligning to current running-config behavior.

Reply:
frr-reload just executes a generated list of commands and doesn't store it anywhere. It doesn't matter whether `exit-vrf` or `exit` is executed. The same thing with other non-standard exits like `exit-address-family` in BGP which doesn't have any specific code. I don't see a point in adding more complicated code that's not really necessary.

Reply:
Ack. it would have been best to keep `exit-vrf` like `exit-address-family` rather than calling simply generic `exit`.  So far this PR addresses few isssues so we will get this in.